gamestonk_terminal/cryptocurrency/coinpaprika/coinpaprika_view.py

```diff
@@ -0,0 +1,1541 @@
+        help="""Flag to show urls. If you will use that flag you will see only:
+        exchange, pair, trust_score, market_url columns""",
+        default=False,
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if not ns_parser:
+            return
+
+        vs = ns_parser.vs
+        sort = ns_parser.sortby
+        if sort in ["volume", "price"]:
+            sort = f"{str(vs).lower()}_{sort}"
+
+        df = paprika.get_coin_markets_by_id(coin_id, vs)
+
+        if df.empty:
+            return
+
+        df = df.sort_values(by=sort, ascending=ns_parser.descend)
+
+        if ns_parser.links is True:
+            df = df[["exchange", "pair", "trust_score", "market_url"]]
+        else:
+            df.drop("market_url", axis=1, inplace=True)
+
+        print(
+            tabulate(
+                df.head(ns_parser.top),
+                headers=df.columns,
+                floatfmt=".2f",
+                showindex=False,
+                tablefmt="fancy_grid",
+            )
+        )
+        print("")
+
+    except Exception as e:
+        print(e)
+        print("")
+
+
+def chart(coin_id: str, other_args: List[str]):
+    """Plots chart for loaded cryptocurrency
+
+    Parameters
+    ----------
+    coin_id: str
+        Identifier of coin for CoinPaprika API
+    other_args: List[str]
+        Arguments to pass to argparse
+
+    """
+    parser = argparse.ArgumentParser(
+        add_help=False,
+        prog="chart",
+        description="""
+                        Display chart for loaded coin. You can specify currency vs which you want
+                        to show chart and also number of days to get data for.
+                        By default currency: usd and days: 90.
+                        E.g. if you loaded in previous step Ethereum and you want to see it's price vs bitcoin
+                        in last 90 days range use `chart --vs btc --days 90`
+                        Available quoted currencies are only btc and usd
+                        """,
+    )
+
+    parser.add_argument(
+        "--vs",
+        default="usd",
+        dest="vs",
+        help="Currency to display vs coin",
+        choices=["usd", "btc", "BTC", "USD"],
+    )
+
+    parser.add_argument(
+        "-d",
+        "--days",
+        default=90,
+        dest="days",
+        type=int,
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+
+        if not ns_parser:
+            return
+
+        df = paprika.get_ohlc_historical(coin_id, ns_parser.vs.upper(), ns_parser.days)
+
+        if df.empty:
+            return
+
+        df.drop(["time_close", "market_cap"], axis=1, inplace=True)
+        df.columns = [
+            "Time0",
+            "Open",
+            "High",
+            "Low",
+            "Close",
+            "Volume",
+        ]
+        df = df.set_index(pd.to_datetime(df["Time0"])).drop("Time0", axis=1)
+        title = (
```

Comment:
Yea, true. But what should be expected input for a title?

Reply:
`        title = f"{coin_id}/{ns_parser.vs} from {df.index[0].strftime('%Y/%m/%d')} to {df.index[-1].strftime('%Y/%m/%d')}"`

Reply:
Exactly. 

So the issue is that you did something like:
```
title = ("This is my title",) 
```
and this is a tuple with 1 string as element.

I'm assuming that when you tried to plot you had an error because it wasn't expecting structures, hence you casted the tuple type to a string using:
```
title_plot = str(title)
```
This leads to converting the entire tuple as a string, hence resulting in what you've got. E.g.

<img width="215" alt="Captura de ecrã 2021-07-14, às 23 22 43" src="https://user-images.githubusercontent.com/25267873/125700684-cd589309-fe30-4164-946f-fe8fccee3be8.png">

What you want is just
```
title_plot = "This is my title"
```